ArduPlane/quadplane.h

```diff
@@ -329,6 +329,9 @@ class QuadPlane
     AP_Int8 enable;
     AP_Int8 transition_pitch_max;
 
+    // landing detection threshold in meters
+    AP_Float land_detect_threshold;
```

Comment:
Maybe put it in the struct

https://github.com/ArduPilot/ardupilot/blob/54208574f0ee96b406a854069d787d34cfa6d5f7/ArduPlane/quadplane.h#L400-L405

Reply:
done